dockerfile

```diff
@@ -0,0 +1,16 @@
+# base image
+FROM node:12.2.0-alpine
+
+# set working directory
+WORKDIR /app
+
+# add `/app/node_modules/.bin` to $PATH
+ENV PATH /app/node_modules/.bin:$PATH
+
+# install and cache app dependencies
+COPY package.json /app/package.json
+RUN npm install
+RUN npm install @vue/cli@3.7.0 -g
```

Comment:
@vue/cli hoeft volgens mij niet globaal geinstalleerd te worden omdat de cli-service als dependency al in het project zit.

Reply:
Klopt.